Userland/Libraries/LibCore/Stream.h

```diff
@@ -113,8 +113,8 @@ class Stream {
     ErrorOr<T> read_trivial_value()
     {
         alignas(T) u8 buffer[sizeof(T)] = {};
-        TRY(read_entire_buffer(buffer));
-        return *bit_cast<T>(buffer);
+        TRY(read_entire_buffer({ &buffer, sizeof(buffer) }));
+        return *bit_cast<T*>(&buffer);
```

Comment:
This seems like a more complicated way of just doing:
```c++
return bit_cast<T>(buffer);
```

Reply:
Quite possible, I'm not all too familiar with C++ casts. I'll check whether the direct cast works as well...